core/src/com/unciv/models/ruleset/GameSpeed.kt

```diff
@@ -22,6 +25,16 @@ class GameSpeed : RulesetObject() {
     override fun getUniqueTarget(): UniqueTarget = UniqueTarget.Speed
 
     override fun makeLink(): String = "GameSpeed/$name"
+    override fun getCivilopediaTextHeader() = FormattedLine(name, header = 2)
+    override fun getCivilopediaTextLines(ruleset: Ruleset) = sequence {
+        yield(FormattedLine("General speed modifier: [${modifier * 100}]% ${Fonts.turn}"))
```

Comment:
Doesn't quite match your template - this way the Fonts symbol (and the blank) becomes part of the key. Flexible for translators, more  work for us. The opposite would be `"{General speed modifier}: ${modifier * 100}% ${Fonts.turn}"` and just "General speed modifier" as translation key, but xlators would have no control over the rest of the line... Your choice, but try a quick test with a lang of your choice.

Reply:
I haven't had the fortune of working with translations yet. Are my changes just now what you were meaning? I tried the new translation.properties with a French translation of the new strings and it seems to work.